editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -409,9 +401,13 @@ function trimWhitespaces(
   return trimmedText
 }
 
+export var Fragment: React.FunctionComponent<React.PropsWithChildren<any>> = ({ children }) => {
```

Comment:
why not just 
`const Fragment = React.Fragment` ?

Reply:
Good idea, then we don't even have to create a local Fragment const, just use the one imported from React

Reply:
Fix in https://github.com/concrete-utopia/utopia/pull/3273